Clarify callout options and update to markdown from Liquid

emangulo opened the pull request:
### Why:
In order to better align to the style guide and improve consistency in syntax.

<!-- If there's an existing issue for your change, please link to it above.
If there's _not_ an existing issue, please open one first to make it more likely that this update will be accepted: https://github.com/github/docs/issues/new/choose. -->

### What's being changed (if available, include any code snippets, screenshots, or gifs):

- Updated the **Basic writing and formatting syntax** file to have the callouts be in markdown syntax instead of Liquid syntax.
- Updated the "Alerts" section of the same file to "Callouts" to align on the naming convention used in the style guide.
- Updated the **Style guide** file to include Important as a callout option as described in the **Basic writing and formatting syntax** file

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

emangulo committed:
Update callouts to markdown

Update callouts to markdown syntax from Liquid syntax to align with the style guide.

emangulo committed:
Update Alert to Callout title

Updated the Alert section to be the Callout section in order to align to what the GitHub style guide calls them.

emangulo committed:
Add Important callout as an option

Include Important callout as an option as described in the Basic writing and formatting syntax file

emangulo committed:
Update callout to markdown syntax

nguyenalex836 commented:
@emangulo Thanks so much for opening a PR! I'll get this triaged for review ✨

felicitymay commented:
@emangulo - many thanks for flagging this discrepancy between the description of how to use Notes etc. in GitHub generally, and on the style guide for contributors to the user documentation for GitHub. I think you're right that we should use the same term for both.

I'm going to check in with our content strategy team on whether we should use the term "alerts" or "callouts" for both before I review this PR.

ethanpalm commented on content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md:
```diff
@@ -328,23 +305,18 @@ The footnote will render like this:
+## Callouts
 
-Alerts are a Markdown extension based on the blockquote syntax that you can use to emphasize critical information. On {% data variables.product.product_name %}, they are displayed with distinctive colors and icons to indicate the significance of the content.
+Callouts are a Markdown extension based on the blockquote syntax that you can use to emphasize critical information. On {% data variables.product.product_name %}, they are displayed with distinctive colors and icons to indicate the significance of the content.
```
```suggestion
Alerts are a Markdown extension based on the blockquote syntax that you can use to emphasize critical information. On {% data variables.product.product_name %}, they are displayed with distinctive colors and icons to indicate the significance of the content.
```

ethanpalm commented on content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md:
```diff
@@ -328,23 +305,18 @@ The footnote will render like this:
 {% ifversion markdown-alerts %}
 
-## Alerts
+## Callouts
```
```suggestion
## Alerts
```

ethanpalm commented on content/contributing/style-guide-and-content-model/style-guide.md:
```diff
@@ -47,16 +44,7 @@ Callouts should be concise. If the information consists of more than a couple of
 
 ### Callout types
```
```suggestion
### Alert types
```

ethanpalm commented on content/contributing/style-guide-and-content-model/style-guide.md:
```diff
@@ -47,16 +44,7 @@ Callouts should be concise. If the information consists of more than a couple of
-
-> [!TIP]
-> When you @mention an organization, only those that you're a member of will autocomplete. You can still @mention organizations that you're not a member of, like a previous employer, but the organization name won't autocomplete for you.
+There are five types of callouts: Note, Tip, Important, Warning, and Caution.
```
```suggestion
We use four types of callouts: Note, Tip, Warning, and Caution.
```

ethanpalm commented on content/contributing/style-guide-and-content-model/style-guide.md:
```diff
@@ -71,6 +59,20 @@ For example, "[AUTOTITLE](/code-security/secret-scanning/managing-alerts-from-se
+> [!TIP]
+> When you @mention an organization, only those that you're a member of will autocomplete. You can still @mention organizations that you're not a member of, like a previous employer, but the organization name won't autocomplete for you.
+
+#### Important
+
+> [!IMPORTANT]
+> Key information users need to know to achieve their goal.
+
```
```suggestion
```

ethanpalm commented on content/contributing/style-guide-and-content-model/style-guide.md:
````diff
@@ -94,18 +96,25 @@ We use standard formatting and colors for different types of callouts across doc
 ```
 
-Note:
+Important:
 
````
```suggestion
```

ethanpalm commented on content/contributing/style-guide-and-content-model/style-guide.md:
````diff
@@ -94,18 +96,25 @@ We use standard formatting and colors for different types of callouts across doc
 ```markdown
-> [!NOTE]
-> Keep this in mind.
+> [!IMPORTANT]
+> This is important.
 ```
````
```suggestion
```

ethanpalm commented on content/contributing/style-guide-and-content-model/style-guide.md:
````diff
@@ -94,18 +96,25 @@ We use standard formatting and colors for different types of callouts across doc
-Note:
+Important:
 
 ```markdown
````
```suggestion
```

ethanpalm reviewed (commented):
Hi @emangulo, thanks for creating this PR improve the consistency in syntax ✨ .

We should definitely use a single term, but since github.com uses "Alerts" and "Callouts" is just a GitHub Docs-specific term, we should bring things into alignment using "Alerts." I've added some suggestions to make those changes and I'll push a commit for the other paragraphs in the style guide that should be updated.

And while Important alerts are supported, we avoid using them in the Docs. I've suggested another change to make that more clear.

Thanks for highlighting the changes that needed to be made for these files and collaborating. Once the suggestions are incorporated, I can get this merged into the Docs 😄 .

emangulo committed:
Align style guide to alert instead of callout

Co-authored-by: Ethan Palm <56270045+ethanpalm@users.noreply.github.com>

emangulo committed:
Update alert section in style guide

Updated all references to callouts in the "Alert" section to "alert"

emangulo committed:
Apply suggestions. Remove Important alert from style guide

Co-authored-by: Ethan Palm <56270045+ethanpalm@users.noreply.github.com>

emangulo committed:
Remove double blank line for linter

emangulo committed:
Update callout section to alert section

emangulo commented:
Hi @ethanpalm,
I have included your suggestions and also changed any additional references to callouts in the Alerts section of the style guide. Also removed the Important examples.

I have also updated this [Callout tags](https://docs.github.com/en/contributing/writing-for-github-docs/using-markdown-and-liquid-in-github-docs#callout-tags) section to _Alerts_

ethanpalm reviewed (dismissed):
These changes look great, @emangulo. I'll get the branch up to date and make sure all checks are passing then merge the PR. Thank you for your contribution to the GitHub Docs!

ethanpalm committed:
Merge branch 'main' into main

ethanpalm commented on content/contributing/writing-for-github-docs/using-markdown-and-liquid-in-github-docs.md:
```diff
@@ -48,29 +48,29 @@ This content is displayed on the {% data variables.product.prodname_docs %} site
+Alerts highlight important information that users need to know. We use standard formatting and colors for four different types of Alerts: Note, Tip, Warning, and Caution.
 
-For information on when to use callouts, and how to format them in Markdown, see "[AUTOTITLE](/contributing/style-guide-and-content-model/style-guide#callouts)."
+For information on when to use alerts, and how to format them in Markdown, see "[AUTOTITLE](/contributing/style-guide-and-content-model/style-guide#callouts)."
```
```suggestion
For information on when to use alerts, and how to format them in Markdown, see "[AUTOTITLE](/contributing/style-guide-and-content-model/style-guide#alerts)."
```

ethanpalm commented on content/contributing/style-guide-and-content-model/style-guide.md:
```diff
@@ -122,9 +119,9 @@ Caution:
+Liquid syntax for alerts is still supported and may still appear in older articles, but should not be used for new alerts.
 
-For more information on formatting callouts, see “Callouts” in "[AUTOTITLE](/contributing/syntax-and-versioning-for-github-docs/using-markdown-and-liquid-in-github-docs#callout-tags)."
+For more information on formatting alerts, see “Alerts” in "[AUTOTITLE](/contributing/syntax-and-versioning-for-github-docs/using-markdown-and-liquid-in-github-docs#callout-tags)."
```
```suggestion
For more information on formatting alerts, see “Alerts” in "[AUTOTITLE](/contributing/syntax-and-versioning-for-github-docs/using-markdown-and-liquid-in-github-docs#alerts)."
```

ethanpalm committed:
Update link anchors

ethanpalm committed:
Update link anchor

ethanpalm reviewed (approved):
There was an anchor in the footnotes section that was causing a failing test. Fixed and ready to merge now
